Jenkinsfile

```diff
@@ -25,6 +25,9 @@ pipeline {
 				dir ('examples') {
 					sh "$MVN deploy"
 				}
+                dir ('jaxrs-spec') {
```

Comment:
The file is currently using tabs for indentation and you added spaces. I prefer spaces, but let's not mix them.

Reply:
Absolutely +1, but can somebody tell me how I can tell [Gitpod](https://gitpod.io/#https://github.com/eclipse-ee4j/jaxrs-api/pull/808) to keep the tabs...? I type tabs, the editor shows tabs, but in the file are blanks. ;-(

Reply:
Solved. Used `vi`. 40 years old technoligy works better than 1 year old. ;-)